README.md

```diff
@@ -103,13 +103,16 @@ client := hystrix.NewClient(
 	hystrix.WithHystrixTimeout(1000 * time.Millisecond),
 	hystrix.WithMaxConcurrentRequests(30),
 	hystrix.WithErrorPercentThreshold(20),
+	hystrix.WithStatsDCollector("localhost:8125", "myapp.hystrix"),
 )
 
 // The rest is the same as the previous example
 ```
 
 In the above example, there are two timeout values used: one for the hystrix configuration, and one for the HTTP client configuration. The former determines the time at which hystrix should register an error, while the latter determines when the client itself should return a timeout error. Unless you have any special requirements, both of these would have the same values.
 
+You can choose to export hystrix metrics to a statsD collector with the `hystrix.WithStatsDCollector(<statsd addr>, <metrics-prefix>)` option when initializing the client as shown above.
```

Comment:
Can we add this in `Plugins` section? Check the current documentation.

Rest all LGTM

Reply:
This is not implemented as a plugin but as core functionality that can be activated via option methods. Do you recommend this to be moved to plugins nonetheless?

Reply:
Was thinking it would be cleaner to implement it as a plugin. Wdyt?

Reply:
hmm, currently we are just using the hystrix-go plugin for this `plugins.InitializeStatsdCollector(client.statsD)`.   
Implementing it as a heimdall plugin would mean registering an in-memory collector with hystrix-go get the data, then triggering the statsd publish on request `OnRequestEnd`.  
Not ideal for 2 reasons:
- we will have to write the in-memory plugin code in heimdall
- the metrics data publishing is not tied to the request flow, but time - so the heimdall plugin interface isn't a good fit for this imo
```go
type Plugin interface {
	OnRequestStart(*http.Request)
	OnRequestEnd(*http.Request, *http.Response)
	OnError(*http.Request, error)
}
```